jumpscale/sals/nettools/__init__.py

```diff
@@ -375,98 +459,240 @@ def get_host_by_name(dnsHostname: str):
     return socket.gethostbyname(dnsHostname)
 
 
-def ping_machine(ip, pingtimeout=60, allowhostname=True):
+def ping_machine(ip: str, timeout: Optional[int] = 60, allowhostname: Optional[bool] = True) -> bool:
```

Comment:
in here we change the method signature, we should search for all it's usage and replace it if it's used anywhere

Reply:
I checked and it is not used in js-ng and js-sdk, not sure if I should search elsewhere.

Reply:
great